AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/ReconcileAndLocalSave/ModelReconciliationQueue.swift

```diff
@@ -12,6 +12,7 @@ import Combine
 enum ModelReconciliationQueueEvent {
     case started
     case paused
+    case finished
```

Comment:
Is this event only thrown after an initial sync is finished? What about after the initial sync, when the reconciliation queue is receiving subscription updates?